README.md

```diff
@@ -31,6 +31,7 @@ system_packages:
   - musl
   - make
   - g++
+  - python3=3.8.2-r1
```

Comment:
```suggestion
  - python3
```

I prefer not fixing the dependency versions.

I've tested without it, and it also fixes the problem.